@xen-orchestra/web/src/components/account-menu/AccountMenu.vue

```diff
@@ -3,6 +3,26 @@
     <template #trigger="{ isOpen, open }">
       <AccountMenuTrigger :active="isOpen" @click="open($event)" />
     </template>
+    <MenuItem :icon="faBook">
```

Comment:
If not already done, we should create a Kanban card to update `MenuItem` so that it can accept links directly.